### DISCUSSION TITLE
Help converting GENDF file to ace file or pendf/endf file.

### DISCUSSION THREAD

**Discussion by SSYIMP** [njoy/NJOY21#177] [Q&A]
NJOY community:
I'm trying to transform the  GENDF file generated by the GROUPR in njoy21 into an ACE file, but I did not find a suitable method in the manual. 
Taking Fe56 as an example, the input file for njoy21 is as follows: tape20 is the ENDF file for Fe56 (https://www-nds.iaea.org/ads/ENDF/FE_056.ZIP), and tape91 is the  GENDF file. Can you help me achieve file format conversion？

moder/ Extract/convert neutron evaluated data/endf tape
1 21
'ENDF for 26-Fe-56'
20 2631
0/
reconr/ Reconstruct XS for neutrons/endf-pendf
21 22
'PENDF for 26-Fe-56'/
2631 1 31/ ngrid
0.001/
'Processed by NJOY2021'/
1e4 2e4 3e4 4e4 5e4 6e4 7e4 8e4 9e4 
1e5 2e5 3e5 4e5 5e5 6e5 7e5 8e5 9e5
1e6 2e6 3e6 4e6 5e6 6e6 7e6 8e6 9e6
1e7 2e7 3e7 4.987654e7/ energy grid--eV
0/
broadr/ Doppler broaden XS/pendf-pendf
21 22 23
2631 1/  
0.001 2.0e7/
293.6
0/
heatr / Add heating kerma and damage energy/pendf-pendf
21 23 24 90/
2631 3 0 0 0 2 0 /
442 443 444/
unresr/ self-shielded /pendf-pendf
21 24 25
2631 1 16/   sigma_zero
293.6
1e10 5e3 1e3 1e2 50 20 10 
5 2 1 0.5 0.2 0.1 0.05 0.01 0.001/ sigma_vaules
0/
purr/ self-shielded /pendf-pendf
21 25 26
2631  1 16 20 32/
293.6/
1e10 5e3 1e3 1e2 50 20 10 
5 2 1 0.5 0.2 0.1 0.05 0.01 0.01/ sigma_vaules
0/
groupr/ pendf-gendf
21 26 0 91
2631 3 2 8 0 1 16 1/
'26-Fe-56 gendf'/
293.6/
1e10 5e3 1e3 1e2 50 20 10 
5 2 1 0.5 0.2 0.1 0.05 0.01 0.01/ sigma_vaules
3/
0/
0/
stop

**Comment by nathangibson14** [comment]
Hello,

If you are trying to create continuous energy ACE files using the same data that was used to make the multigroup GENDF files, your issue is that GROUPR output does not go into ACER.  Instead, for your example, you'd use tape 21 as the ENDF file and tape 26 (output of PURR) as the PENDF file for making the ACE file.  You can think of GROUPR and ACER as being parallel paths branching out from a common starting point.

If you are instead trying to make ACE files for multigroup Monte Carlo, such files do exist, but only informally.  NJOY does not support this operation.

I hope that helps.

--Nathan

**Reply by SSYIMP** [reply]
Hello Nathan,
What is the "informal method" mentioned in your response? Can you introduce it. I hope that through this type of method, GENDF file can be transformed into ACE files that can be recognized by MNCPX or other MC software.
--Shiyu Song
